Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -3009,4 +3006,173 @@ void SoftwareGLContext::sync_device_texcoord_config()
     m_rasterizer.set_options(options);
 }
 
+void SoftwareGLContext::gl_lightf(GLenum light, GLenum pname, GLfloat param)
```

Comment:
This should also work inside of a list

Reply:
I believe I've already added this to the right place for it to be executed in a list??

https://github.com/SerenityOS/serenity/pull/11764/files/0b92a7a66a769f8f875d114d589a51b5f22c6978#diff-0dbbc803292909480f93f2043b97891875af950652570513caa853651184b78aR370

Reply:
You still need `APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED` inside the methods to make this work

Reply:
Whoops!